brave/browser-laptop
# don’t use sync ipc for publisher info

Submitter Checklist:

the diff is mostly an indenting change from wrapping in ipcRenderer.once


Test Plan:
1. Enable ledger
2. Visit some pubs long enough to add them
3. Pubs should show in ledger prefs

Reviewer Checklist:

Tests
